### DISCUSSION TITLE
Landing page template - integration icons get stretched

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2807] [Q&A] [answered]
Hi, the landing page template has some integration icons that, at some breakpoints, get stretched and deformed.

Check the screenshot for medium size. It's not much noticeable for logos but I'm using this for testimonial avatars and it's noticeable.

**Wide**:

![Captura de pantalla 2024-02-01 a las 11 31 17](https://github.com/saadeghi/daisyui/assets/2629/3385afea-55a1-4761-a4fb-7021fa4cfccf)

**Medium**:

![Captura de pantalla 2024-02-01 a las 11 31 27](https://github.com/saadeghi/daisyui/assets/2629/2f419775-1908-4f6a-8e46-67df442e8414)

**Narrow**:

![Captura de pantalla 2024-02-01 a las 11 31 38](https://github.com/saadeghi/daisyui/assets/2629/b0cc292d-3683-4f0f-8dac-269966ed6f54)

**Comment by saadeghi** [comment]
Thanks for reporting. Will be fixed this week ✅

**Comment by saadeghi** [comment, marked as answer]
Fixed in version 1.1.0  
The file is updated and you can download the new version https://app.lemonsqueezy.com/my-orders
